instrumentation/resources/library/src/test/java/io/opentelemetry/instrumentation/resources/ProcessResourceTest.java

```diff
@@ -29,7 +30,7 @@ void notWindows() {
 
     assertThat(attributes.get(ResourceAttributes.PROCESS_PID)).isGreaterThan(1);
     assertThat(attributes.get(ResourceAttributes.PROCESS_EXECUTABLE_PATH))
-        .contains("java")
+        .contains(File.separatorChar + "java")
```

Comment:
maybe something like `.matches("[/\\]java)"`? e.g. to avoid the test verification relying on the same logic that we missed previously

Reply:
I have updated the tests accordingly.